alerts/cluster-etcd-operator/etcdNoLeader.md

```diff
@@ -41,4 +41,4 @@ namespace/project and in the `etcd` container.
 Follow the steps described in the [disaster and recovery docs][docs].
 
 
-[docs]: https://docs.openshift.com/container-platform/4.7/backup_and_restore/disaster_recovery/about-disaster-recovery.html
+[docs]: https://docs.openshift.com/container-platform/4.10/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/about-disaster-recovery.html
```

Comment:
Using `latest` instead of a version will redirect it to the latest y-stream docs.

```suggestion
[docs]: https://docs.openshift.com/container-platform/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/about-disaster-recovery.html
```